catalog-rest-service/src/main/resources/json/schema/operations/pipelines/airflowPipeline.json

```diff
@@ -151,6 +186,70 @@
       },
       "default": null
     },
+    "maxActiveRuns": {
```

Comment:
we could not add it in the JSON schema, but in the REST project it would be interesting to create the DAGs with `catchup=False` to avoid multiple DAG runs and just get the latest one. I don't think we will miss any info this way